src/System.IO.Abstractions.TestingHelpers/MockDirectoryInfo.cs

```diff
@@ -134,7 +134,15 @@ public override DateTime LastWriteTimeUtc
             set { GetMockFileDataForWrite().LastWriteTime = value.ToLocalTime(); }
         }
 
+#if FEATURE_FILE_SYSTEM_INFO_LINK_TARGET
         /// <inheritdoc />
+        public override string LinkTarget
+        {
+            get { return GetMockFileDataForRead().LinkTarget; }
+        }
+#endif
+
+            /// <inheritdoc />
         public override string Name
```

Comment:
```suggestion
        /// <inheritdoc />
        public override string Name
```

Reply:
Oh wow. I'm sorry about this. Not even sure how it happened :man_facepalming: